tests/integ/modin/series/test_loc.py

```diff
@@ -1749,11 +1749,11 @@ def test_series_non_partial_string_indexing_cases(ops, error):
             eval_snowpark_pandas_result(snowpark_str, native_str, ops, check_freq=False)
 
 
-@sql_count_checker(query_count=2)
+@sql_count_checker(query_count=3)
```

Comment:
do you know why now we have one extra query?

Reply:
Fixed.